platform-utils/endpoints-generator/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "endpoints-generator",
+  "main": "index.js",
+  "scripts": {
+    "build": "node index.js"
+  },
+  "private": true,
+  "type": "module",
+  "devDependencies": {
+    "fs": "^0.0.1-security"
```

Comment:
Isn't fs a core library?

Reply:
Also, it's dev for whatever this is used for, but most defnitely a production library locally.